### ISSUE TITLE
Feature request: Add define to change return value from double to long double

### ISSUE THREAD

**Issue by PQCraft** [codeplea/tinyexpr#92]

**Comment by codeplea** [comment]
>Can you tell me if TinyExpr is able to evaluate summation, Σ (sigma) to calculate a sum of multiple terms?

No, not directly. You can do "term1 + term2 + term3".  And it would be easy to add sigma, as it's easy to add any function.

**Comment by codeplea** [comment]
Check out the documentation at: https://github.com/codeplea/tinyexpr#binding-to-custom-functions

You can add in any custom function you like.

**Comment by codeplea** [comment]
Closing, see CONTRIBUTING. TinyExpr is small enough that you can add this yourself in a fork. It only stays that way if I don't implement every request like this one.
